README.md

```diff
@@ -14,28 +14,46 @@ A Linux/macOS environment is required at the moment.
 
 ### Installation steps
```

Comment:
If this is not strictly related to certs, I would commit it separately

Reply:
You mean I should make two separate commits on the same branch, or I should commit the changes of README on a different branch? Indeed, if the certs are contained in the repository, the previous version of the README is not up to date anymore, that's why I added all the changes in a single commit.

Reply:
I mean, two different PRs. This PR should contain only SPID demo certificates, SPID demo configurations and SPID related readmes.
I would propose to
1) Create a PR (to be merged first) with the generic readme (without SPID related steps)
2) Create a PR with SPID certificates, configs and readme deltas